src/components/v2/V2Create/tabs/TokenTabContent.tsx

```diff
@@ -1,39 +1,235 @@
+  )
+
+  const [redemptionRateDisabled, setRedemptionRateDisabled] = useState<boolean>(
+    reduxRedemptionRate === undefined || reduxRedemptionRate === '100',
+  )
 
-  const onTokenFormSaved = useCallback(
-    (fields: TokenFormFields) => {
-      dispatch(editingV2ProjectActions.setDiscountRate(fields.discountRate))
-      dispatch(editingV2ProjectActions.setReservedRate(fields.reservedRate))
-    },
-    [dispatch],
+  // Using a state here because relying on the form does not
+  // pass through updated reservedRate to ProjectTicketMods
+  const [reservedRate, setReservedRate] = useState<number | undefined>(
+    parseFloat(reduxReservedRate),
   )
 
+  const dispatch = useAppDispatch()
+
+  const onTokenFormSaved = useCallback(() => {
+    const fields = tokenForm.getFieldsValue(true)
+    dispatch(editingV2ProjectActions.setDiscountRate(fields.discountRate))
+    dispatch(editingV2ProjectActions.setReservedRate(fields.reservedRate))
+    dispatch(editingV2ProjectActions.setRedemptionRate(fields.redemptionRate))
+    dispatch(editingV2ProjectActions.setReserveTokenSplits(reserveTokenSplits))
+  }, [dispatch, tokenForm, reserveTokenSplits])
+
   const resetTokenForm = useCallback(() => {
     tokenForm.setFieldsValue({
-      discountRate: fundingCycleData?.discountRate ?? '0',
+      discountRate: reduxDiscountRate ?? '0',
+      reservedRate: reduxReservedRate ?? '0',
+      redemptionRate: reduxRedemptionRate ?? '100',
     })
-  }, [fundingCycleData?.discountRate, tokenForm])
+    setReserveTokenSplits(reserveTokenSplits)
+  }, [
+    reduxDiscountRate,
+    reduxReservedRate,
+    reduxRedemptionRate,
+    reserveTokenSplits,
+    tokenForm,
+  ])
+
+  const disableTextStyle: CSSProperties = {
+    color: colors.text.primary,
+    fontStyle: 'italic',
+    fontWeight: 500,
+    marginBottom: 10,
+  }
 
   // initially fill form with any existing redux state
   useEffect(() => {
     resetTokenForm()
   }, [resetTokenForm])
 
+  // Assume the first item is the one of interest.
+  const fundAccessConstraint = fundAccessConstraints[0] as
+    | SerializedV2FundAccessConstraint
+    | undefined
+
+  const _hasFundingTarget = Boolean(
+    parseFloat(fundAccessConstraint?.distributionLimit ?? '0'),
+  )
+  const _hasFundingDuration = Boolean(
+    parseFloat(fundingCycleData?.duration ?? '0'),
+  )
+
   return (
     <div>
       <Space direction="vertical" size="large">
-        <TokenForm form={tokenForm} onSave={onTokenFormSaved} />
+        <p>
+          <Trans>
+            <strong>Note: </strong>Updates you make to token attributes will{' '}
+            <i>not be applied immediately</i> and only take effect in{' '}
+            <i>upcoming funding cycles.</i>
+          </Trans>
+        </p>
+        <Form form={tokenForm} layout="vertical">
+          <FormItems.ProjectReserved
+            value={tokenForm.getFieldValue('reservedRate') ?? reduxReservedRate}
+            name="reservedRate"
```

Comment:
pls remove `name` whereever we have an `onChange` as the antd form binding can cause unexpected behavior.

Reply:
Got it